jadx-gui/src/main/resources/i18n/Messages_de_DE.properties

```diff
@@ -128,7 +128,7 @@ preferences.showInconsistentCode=Inkonsistenten Code anzeigen
 preferences.escapeUnicode=Unicodezeichen escapen
 preferences.replaceConsts=Konstanten ersetzen
 preferences.respectBytecodeAccessModifiers=Modifikatoren für Bytecode-Zugriff beachten
-preferences.useImports=Importanweisungen anwenden
+preferences.useImports=Importierte Anweisungen anwenden
```

Comment:
I still think that people don't understand this preference option if you name it `Importierte Anweisungen anwenden`. Actually it decides if Jadx always uses fully qualified class names in the source code (option disabled) or if `import packagea.ClassNameY` statements are generated at the beginning at each Java source file. 

In my opinion that can be understood by the current translation `Generiere Import statements`  but is more difficult to understand by your translations. Please keep in mind that just translating items is not what makes a good translation. A good translation should name the option it belongs to in a way people using the language understand it. Therefore for a translation it is best to check where it is used, then understand what it actually means and then create the translation.